src/engine/TPA.cc

```diff
@@ -562,10 +550,11 @@ TPASplit::QueryResult TPASplit::reachabilityQueryExact(PTRef from, PTRef to, uns
                     return result;
                 }
                 // Create the three states corresponding to current, next and next-next variables from the query
-//              PTRef modelMidpoint = getNextVersion(extractStateFromModel(getStateVars(1), *model), -1);
+                //              PTRef modelMidpoint = getNextVersion(extractStateFromModel(getStateVars(1), *model),
```

Comment:
This space after start of comment is also weird...
Something to update in clang-format config.